erpnext/hr/doctype/employee/employee_dashboard.py

```diff
@@ -13,23 +13,31 @@ def get_data():
 			},
 			{
 				'label': _('Payroll'),
-				'items': ['Salary Structure Assignment', 'Salary Slip', 'Timesheet']
+				'items': ['Salary Structure Assignment', 'Salary Slip', 'Additional Salary', 'Timesheet']
+			},
+			{
+				'label': _('Shift'),
+				'items': ['Shift Request', 'Shift Assignment']
 			},
 			{
 				'label': _('Expense'),
-				'items': ['Expense Claim']
+				'items': ['Expense Claim', 'Travel Request']
 			},
 			{
 				'label': _('Evaluation'),
 				'items': ['Appraisal']
 			},
-			{
-				'label': _('Training'),
-				'items': ['Training Event', 'Training Result']
-			},
 			{
 				'label': _('Lifecycle'),
 				'items': ['Employee Transfer', 'Employee Promotion', 'Employee Separation']
-			}
+			},
+			{
+				'label': _('Benefit'),
```

Comment:
should be "Payroll"

Reply:
I think there should be a different label name because payroll section already contains many links and is also larger. can we can split it ??

Reply:
**Updated Employee dashboard: **

![Screenshot 2019-06-14 at 2 07 02 PM](https://user-images.githubusercontent.com/32095923/59495857-d8d78480-8ead-11e9-8c72-9370c2d93299.png)